db/fixtures/notification_types.yml

```diff
@@ -269,3 +269,8 @@
   :expires_in: 24.hours
   :level: :error
   :audience: superadmin
+- :name: evm_worker_memory_exceeded
+  :message: 'Killing Worker %{name} using %{memory_usage}mb exceeded limit of %{memory_threshold}mb'
```

Comment:
The wording feels funny...perhaps "Killing worker %{name} due to excessive memory usage.  %{memory_usage} used memory exceeds limit of %{memory_threshold}." ?